services/field-data.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var fc = require('@turf/helpers').featureCollection;
+var map = require('lodash').map;
+var groupBy = require('lodash').groupBy;
+
+/**
+ * @func groupGeometriesById
+ * Given an array of objects that represent field data geometries,
+ * returns an array of objects that, for each road ids, include road geometries grouped by source.
+ * @param geoms {array} array of field data geometries generated by a knex query
+ * @return {array} array of field data geometries grouped by VProMMsId
+ */
+exports.groupGeometriesById = function (geoms) {
```

Comment:
This is ripe for further simplification, but let's first remove the grouping by ID, since that won't be necessary anymore.